src/tools/dev/scripts/bv_support/bv_adios2.sh

```diff
@@ -48,9 +48,8 @@ function bv_adios2_depends_on
             depends_on="$depends_on hdf5"
         fi
 
-        if [[ "$DO_BLOSC" == "yes" ]] ; then
-            # note that at the time of writing (may 2022) ADIOS2 only supports c-blosc1, NOT c-blosc2.
-            depends_on="$depends_on blosc"
+        if [[ "$DO_BLOSC2" == "yes" ]] ; then
```

Comment:
I was under the impression that blosc2 was absolutely required for VisIt's use of adios2.
Will our tests work without an adios2 that was built with blosc2?
If not, then I think blosc2 should be always required, not just when --blosc2 is added to command line. build_visit should complain if adios2 is specified, but blosc2 isn't.

Reply:
I don't think blosc2 is absolutely required for VisIt's use of adios2. I only added the blosc1 (now superseded by blosc2) dependency a couple years ago. However, our tests do require blosc2. Do you think I should make it mandatory?

Reply:
What functionality is lost without blosc2? Are most of our users expecting blosc2 enabled adios2?  I think the answer is yes, in which case the dependency should be mandatory.

Reply:
I'll make it mandatory. I think we lose the ability to read files that use Blosc compression. There are other flavors of files though that can still be read.